spec/core/ics-026-routing-module/UPGRADES.md

```diff
@@ -26,11 +26,11 @@ In order to support channel upgrades, the application must implement the followi
 
 ```typescript
 interface ModuleUpgradeCallbacks {
-    onChanUpgradeInit: onChanUpgradeInit,
-    onChanUpgradeTry: onChanUpgradeTry,
-    onChanUpgradeAck: onChanUpgradeAck,
-    onChanUpgradeConfirm: onChanUpgradeConfirm,
-    onChanUpgradeRestore: onChanUpgradeRestore
+  onChanUpgradeInit: onChanUpgradeInit,
+  onChanUpgradeTry: onChanUpgradeTry,
+  onChanUpgradeAck: onChanUpgradeAck,
+  onChanUpgradeConfirm: onChanUpgradeConfirm,
```

Comment:
```suggestion
  onChanUpgradeOpen: onChanUpgradeOpen,
```

Reply:
Agreed with this since it will be called on both sides